client/state/automated-transfer/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { AUTOMATED_TRANSFER_STATUS_SET } from 'state/action-types';
+import { automatedTransferStatusSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+// Stores the status of automated transfers per site
+export const status = createReducer( {}, {
+	[ AUTOMATED_TRANSFER_STATUS_SET ]: ( state, action ) => ( {
+		...state,
+		[ action.siteId ]: action.automatedTransferStatus,
```

Comment:
Since we don't know anything about what kind of actions will be needed by AT, I believe it's more appropriate to have a single "free" action that can be replaced by whatever will fit best.